src/sql/inspect.py

```diff
@@ -143,14 +153,32 @@ def __init__(self, table_name, schema=None) -> None:
 
                 table_stats[column]["min"] = result_value_values[0][0]
                 table_stats[column]["max"] = result_value_values[0][1]
-                table_stats[column]["unique"] = result_value_values[0][2]
```

Comment:
Since we're not supporting `%sqlcmd` actions at this point - this change is unnecessary. However, I tried to profile a `QuestDB` table, and it failed to retrieve unique values so I removed it from here to a different query (line 165). This means if for any reason our `unique` query fails - we still may see `min`, `max`, and `count` stats. I guess it's good for other DBs as well.